client/state/sharing/services/schema.js

```diff
@@ -0,0 +1,33 @@
+export const itemSchema = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'[a-z]': {
```

Comment:
This will only match properties which have a single character. Also, do we need it to be so strict? For example, we have `google_plus` as a supported service (note underscore).